halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/config/v1/HalconfigDirectoryStructure.java

```diff
@@ -35,7 +34,12 @@ public static void setDirectoryOverride(String directory) {
     directoryOverride.set(directory);
   }
 
-  @Autowired @Setter String halconfigDirectory;
+  private final String halconfigDirectory;
+
+  @Autowired
```

Comment:
NIT: - allargs lombok instead of manual constroctor

Reply:
https://github.com/spinnaker/halyard/pull/2172/commits/70f416336c9ccdb45954c7945607adb87607a0d4